css-grid_place-elements/css/styles.css

```diff
@@ -0,0 +1,82 @@
+/*
+Use grid to fulfill the following tasks:
+
+[x ] - .element__1 should start at column three and go until column five
+[x ] - .element__2 should start at column five and go until the end
+[x ] - .element__5 should span two columns and two rows
+[x ] - .element__8 should span two rows
+[x ] - .element__fish should be in row one and span the entire grid width
+[x ] - .element__20 should start at row six and span two rows
+
+*/
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: system-ui;
+  background-color: #f7fdff;
+}
+
+.container {
+  display: grid;
+  gap: 20px;
+  grid-template-columns: repeat(5, 1fr 2fr);
+  grid-template-rows: repeat(10, 50px);
+  padding: 10px;
+}
+
+.container__element {
+  padding: 10px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: rgb(186, 234, 246);
+  border-radius: 3px;
+  font-size: 2rem;
+}
+.element-1 {
+  background-color: #3a67a4;
+  /* Add styles here */
+  grid-column-start: 3;
+  grid-column-end: 5;
+}
+
+.element-2 {
+  background-color: #3a67a4;
+  /* Add styles here */
+  grid-column-start: 5;
+  grid-column-end: 10;
```

Comment:
Try to use `-1` here to specify the end (regardless of how many columns there are). :slightly_smiling_face:

Reply:
Thank you very much for pointing this out!! I corrected it now.